package.json

```diff
@@ -40,22 +40,24 @@
     "babel-preset-stage-0": "^6.24.1",
     "babel-register": "^6.24.1",
     "chai": "^4.0.2",
+    "enzyme": "^2.9.1",
     "es6-promise": "^4.1.0",
     "eslint": "^4.0.0",
     "eslint-plugin-react": "^7.1.0",
     "mocha": "^3.4.2",
     "npm-install-peers": "^1.1.0",
     "react": "^15.6.1",
+    "react-dom": "^15.6.1",
     "react-redux": "^5.0.5",
+    "react-test-renderer": "^15.6.1",
     "redux": "^3.7.1",
+    "sinon": "^2.3.8",
     "webpack": "^3.0.0",
     "webpack-node-externals": "^1.6.0"
   },
-  "peerDependencies": {
-    "history": "^4.6.3"
-  },
   "dependencies": {
     "lru-memoize": "^1.0.2",
-    "query-string": "^4.3.4"
+    "query-string": "^4.3.4",
+    "history": "^4.6.3"
```

Comment:
Shouldn't. `history` may be needed to run the tests but shouldn't be added to the bundle.
Try installing the peerDeps instead.